### PR TITLE
Add modelingtoolkitize

### PR DESCRIPTION
**Pull request by YingboMa**

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -84,3 +84,27 @@ function vars!(vars, O)
 
     return vars
 end
+
```
Comment (ChrisRackauckas) [inline src/utils.jl]: This should probably go with the ODESystem stuff.

**Comment by HarrisonGrodin** [comment]
Can we name this differently, please? Haha. Maybe `symbolic` or something?

**Comment by ChrisRackauckas** [comment]
`symbolify`? `symbolicitizationifyification`

**Comment by ChrisRackauckas** [comment]
ehh I actually like the name, since most others will clash with libraries like SymPy and whatnot, so we might as well say how we are symbolicing it.

**Pull request merged**